src/main/java/mate/academy/spring/mapper/impl/response/UserResponseMapper.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.spring.mapper.impl.response;
+
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserResponseMapper implements DtoResponseMapper<UserResponseDto, User> {
+    @Override
+    public UserResponseDto toDto(User object) {
+        UserResponseDto responseDto = new UserResponseDto();
```

Comment:
I would change the name of the variable to user to improve the naming, but that's up to you
the same in other classes